src/snaskii.c

```diff
@@ -53,6 +53,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define N_INTRO_SCENES 485 /* Number of frames of the intro animation.*/
 #define N_GAME_SCENES 1    /* Number of frames of the gamepay scnene. */
 
+#define MAX_DELAY 999999 /* Maximum possible delay. */
```

Comment:
Have you tested delay with this value?

Reply:
I tested. I got this value by reading the man page [nanosleep(2)](https://man7.org/linux/man-pages/man2/nanosleep.2.html), there it says that the maximum value accepted is 999999999, given that the game_delay is in microsecond i divided it by 10e3